### ISSUE TITLE
All prefs lost on installation, how to get back?

### ISSUE THREAD

**Issue by jtheletter** [lra/mackup#2035]
I thought this was supposed to back up all my prefs, but instead it reset them all to defaults. How do I get them back? Where did I go wrong?

From the README, all I did was run `brew install mackup` and `mackup backup`, exactly as indicated in the first and only two instructions.

**Comment by jussapaavo** [comment]
Yes, mackup doens't work correctly in Macos Sonoma, since it doesn't support symlinked files for preferences as usual. For more information, see #1924.

**Comment by jtheletter** [comment]
This app should be removed from GitHub, or at least its destruction of all app and OS preferences should be clearly indicated as a warning to future users atop the readme.

**Comment by chrisbloom7** [comment]
> Yes, mackup does not work correctly in Macos Sonoma, since it doesn't support symlinked files for preferences as it previously. For more information, see [#1924](https://github.com/lra/mackup/issues/1924) and especially [this comment](https://github.com/lra/mackup/issues/1924#issuecomment-2115757100).

😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 

Um, maybe that should be communicated via the app itself!?!?! I just lost all my configs after doing a backup - I even ran with the dry-run option first to make sure it was going to add what I expected. Rebooted this morning, all preferences are gone and iCloud does not keep backups.

Yeah, this is an egregious abuse of user trust.

**Comment by jtheletter** [comment]
Here is a pull request to add a warning to users. 

https://github.com/lra/mackup/pull/2043

**Comment by chrisbloom7** [comment]
FYI: [This workaround](https://github.com/lra/mackup/issues/1924#issuecomment-1756330534) worked for me to restore the missing preferences even though iCloud was not showing all the preferences I was expecting to see backed up. 😌

**Comment by chrisbloom7** [comment]
@simphide As of Mackup `v0.8.41`, it is affecting **macOS Sonoma and all later versions**. The bug wasn't something introduced into Mackup but rather the result of a change that Apple made to security settings for the underlying filesystem that caused Mackup's existing functionality to no longer work as expected. As this was a deliberate change by Apple it seems unlikely that a fix will come from their side, so Mackup itself would need to be patched to work around the new restrictions.
